docs/requirements.txt

```diff
@@ -1,6 +1,15 @@
 ablog
-numpydoc
+numpydoc==1.0.0
 recommonmark
-sphinx_rtd_theme
+sphinx_rtd_theme==0.5.0rc1
 sphinx-markdown-tables
-Sphinx==2.2.1
+Sphinx==3.0.4
+nltk
+numpy
+pymongo
+cython
+easydict
+hyperdash
+tensorflow-gpu==2.0.0
```

Comment:
tensorflow-gpu is not necessary for building the doc.

Reply:
or just tensorflow?
sphinx need to execute the .py source file to  build doc, so it have to install tensorlayer and tensorfow at first?

Reply:
you can install `tensorflow` instead of `tensorflow-gpu` here.